### PR TITLE
Adding hover effect on ObjectBrowserNav icon

### PR DESCRIPTION
**Pull request by iFlameing**
screenshot

Screenshot 2021-05-13 at 7 48 20 PM

### CODE REVIEW

File: src/components/manage/Sidebar/ObjectBrowserNav.jsx [hunk 1]
```diff
@@ -90,7 +90,13 @@ const ObjectBrowserNav = ({
               {item.title}
             </span>
             {item.is_folderish && mode === 'image' && (
-              <Icon name={rightArrowSVG} size="24px" />
+              <span>
```
Comment (tisto) [inline src/components/manage/Sidebar/ObjectBrowserNav.jsx:93]: @iFlameing was this span element added by accident? If we change the HTML structure we might have to consider this as a breaking change. I don't see the use of this span element at first, could you elaborate?

Comment (sneridagh) [reply src/components/manage/Sidebar/ObjectBrowserNav.jsx:93]: +1 seems unrelated or a left over?

Comment (iFlameing) [reply src/components/manage/Sidebar/ObjectBrowserNav.jsx:93]: sorry!  I added this to increase the hover effect's space, but it doesn't look good as per UI and I forget to remove it. I will fix it today.

**Comment by sneridagh** [comment]
Once selected, the hover on the arrow (which works in master) is gone. Can we have both?

**Comment by iFlameing** [comment]
@sneridagh tried to fix it please review it.

**Review by sneridagh — APPROVED** [review]
LGTM!
image

**Pull request merged**